docs/syntax.md

```diff
@@ -19,6 +19,7 @@ Below you will find the step syntax next to the name of the method it utilizes.
 - `<GK> [I] (create|submit|delete|update|upsert) [the] resource <non-whitespace-characters>, the operation should (succeed|fail)` kdt.KubeClientSet.ResourceOperationWithResult
 - `<GK> [I] (create|submit|delete|update|upsert) [the] resource <non-whitespace-characters> in [the] <any-characters-except-(")> namespace, the operation should (succeed|fail)` kdt.KubeClientSet.ResourceOperationWithResultInNamespace
 - `<GK> [the] resource <any-characters-except-(")> should be (created|deleted)` kdt.KubeClientSet.ResourceShouldBe
+- `<GK> [the] resource <any-characters-except-(")> should be ready` kdt.KubeClientSet.ResourceShouldBeReady
```

Comment:
Can we not achieve this already using the step below?
`<GK> [the] resource <any-characters-except-(")> condition <any-characters-except-(")> should be <any-characters-except-(")> kdt.KubeClientSet.ResourceConditionShouldBe`